src/components/WeatherCard/WeatherCard.jsx

```diff
@@ -0,0 +1,58 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { useOnboardingContext } from "../../context/user-context";
+import "../WeatherCard/weathercard.css";
+
+export default function WeatherCard() {
+  const [temperature, setTemperature] = useState("");
+  const [userLocation, setUserLocation] = useState("");
+  const { userDetails } = useOnboardingContext();
+  const [temp, tempStatus] = useState(false);
+  const { userInfo, onboardingStatus } = userDetails;
+
+  useEffect(() => {
+    if (onboardingStatus) {
+      const getCurrentTemperature = async () => {
+        try {
+          if (typeof userInfo === "object") {
+            const stringyObj = JSON.stringify(userInfo);
+            const userObj = JSON.parse(stringyObj);
+            setUserLocation(userObj.city);
+          } else {
+            const userObj = JSON.parse(userInfo);
+            setUserLocation(userObj.city);
+          }
+          const coordinatesLink = `http://api.openweathermap.org/geo/1.0/direct?q=${userLocation}&limit=5&appid=27a2ee89e03b4d0b5199b633e513ea3d`;
```

Comment:
Instead of api call you can simply use navigator object as below.

![image](https://user-images.githubusercontent.com/72279128/169984655-52506593-9094-4b0b-be92-166e6e16b66a.png)